contracts/governance/Vesting/Vesting.sol

```diff
@@ -101,15 +105,19 @@ contract Vesting is Ownable{
                 staking.withdraw(stake, i, receiver);
         }
     }
-    
+
+    /**
+     * @dev collect dividends from fee sharing proxy
+     */
     function collectDividends() public onlyOwners{
```

Comment:
The fees should be withdrawn to the token owner

Reply:
maybe we can add a receiver to the withdraw function of the fee sharing proxy?

Reply:
Sorry, I can't correctly work with 2 different branches. I could change it in IFeeSharingProxy, but how it'll be implemented I don't know.
I'll add @todo in IFeeSharingProxy for now.

Reply:
Done. Changed to this signature:
function withdrawToken(address _token, uint32 _maxCheckpoints, address _receiver) public;.